### DISCUSSION TITLE
Posts are not in timestamp order for global timeline

### DISCUSSION THREAD

**Discussion by iamperfor** [pixelfed/pixelfed#5148] [Q&A] [answered]
I recently started my Pixelfed adventure on the Pxlmo instance and the global timeline posts are not showing up in the correct order. The instance shows v0.11.12

edit: Added timestamp details **
**
ie: first post shows 4h and then second post shows a post from Feb 18 and then third post shows 7m ago

Seems like a random order to me

According to pixelfed description -> chronological feeds

![pixelfedfeed](https://github.com/pixelfed/pixelfed/assets/159505379/aac9e35d-7522-40f8-8a8b-45a25e54b634)

Did the app change to a different format now?

**Comment by trwnh** [comment, marked as answer]
"Chronological order" is by the datetime the posts come in, not the datetime the posts claim to be created. This is because someone can set a `created` date that is far in the future and thus be pinned to the top of the timeline.

**Reply by hgdrn** [reply]
Thanks for the explanation.

The sorting of my timeline is a complete mess due to the struggle with the import of Instagram content that imported in a quite random manner. And I had to retry the import some dozens of times during some weeks until I successfully imported all of the 1,600 media.

May be it was possible to differentiate between old content (sort content from the past by timestamp) and future content (sort by creation timestamp) to avoid this pinning?

**Comment by iamperfor** [comment]
> "Chronological order" is by the datetime the posts come in, not the datetime the posts claim to be created. This is because someone can set a `created` date that is far in the future and thus be pinned to the top of the timeline.

Hello I'm new to Pixelfed and thank you for the explanation

-since Pixelfed advertises chronological feeds, what is it referring to?
-does it mean Pixelfed is unable to handle the sorting with the current version?
